docs/literate/src/files/behind_the_scenes_simulation_setup.jl

```diff
@@ -0,0 +1,231 @@
+
+coordinates_min = (-2.0, -2.0)
+coordinates_max = (2.0, 2.0)
+
+coarsening_patches = ((type = "box", coordinates_min = [0.0, -2.0],
+                       coordinates_max = [2.0, 0.0]),)
+
+mesh = TreeMesh(coordinates_min, coordinates_max, initial_refinement_level = 2,
+                n_cells_max = 30_000,
+                coarsening_patches = coarsening_patches)
+
+# The created `TreeMesh` looks like the following:
+
+# ![TreeMesh_example](https://github.com/trixi-framework/Trixi.jl/assets/119304909/d5ef76ee-8246-4730-a692-b472c06063a3)
+
+# Instantiate a [`DGSEM`](@ref) solver with a user-specified polynomial degree. The solver
+# will define `polydeg + 1` Gauss-Lobatto nodes and their associated weights within
+# the reference interval ``[-1, 1]`` in each spatial direction. These nodes will be subsequently
+# used to approximate solutions on each leaf cell of the `TreeMesh`.
+
+solver = DGSEM(polydeg = 3)
+
+# Gauss-Lobatto nodes with `polydeg = 3`:
+
+# ![Gauss-Lobatto_nodes_example](https://github.com/trixi-framework/Trixi.jl/assets/119304909/1d894611-801e-4f75-bff0-d77ca1c672e5)
+
+# ## Overview of the [`SemidiscretizationHyperbolic`](@ref) type
+
+# At this stage, all necessary components for configuring the spatial discretization are in place.
+# The remaining task is to combine these components into a single structure that will be used
+# throughout the entire solving process. This is where [`SemidiscretizationHyperbolic`](@ref) comes
+# into play.
+
+semi = SemidiscretizationHyperbolic(mesh, equations, initial_condition_convergence_test,
+                                    solver)
+
+# The constructor for the `SemidiscretizationHyperbolic` object calls numerous sub-functions to
+# perform the necessary initialization steps. A brief description of the key sub-functions is
+# provided below.
+
+# - `init_elements(leaf_cell_ids, mesh, equations, dg.basis, RealT, uEltype)`
+
+#   The fundamental elements for approximating a solution are the leaf
+#   cells. This implies that on each leaf cell and in each spatial direction, the solution is
+#   treated as a polynomial of the degree specified in the `DGSEM` solver and evaluated at the
+#   Gauss-Lobatto nodes, which were previously illustrated. The `init_elements` function extracts
+#   these leaf cells from the `TreeMesh`, assigns them the label "elements", records their
+#   coordinates, and maps the Gauss-Lobatto nodes from the 1D interval ``[-1, 1]`` onto each axis
+#   of every element.
+
+#   ![elements_example](https://github.com/trixi-framework/Trixi.jl/assets/119304909/9f486670-b579-4e42-8697-439540c8bbb4)
+
+#   The visualization of elements with nodes shown here includes spaces between elements, which do
+#   not exist in reality. This spacing is included only for illustrative purposes to underscore the
+#   separation of elements and the independent projection of nodes onto each element.
+
+# - `init_interfaces(leaf_cell_ids, mesh, elements)`
```

Comment:
When reading through this very nice tutorial, I just realized that this list item has little space to the preceding line but a lot before the following line. Similar places are e.g. here
https://host-docs-from-fork-pr.lakemper.eu/tutorials/behind_the_scenes_simulation_setup.html#Overview-of-the-[semidiscretize](@ref)-function (wrap_array)
Can this somehow be change with markdown?

Reply:
Thank you for the review!
Good point... I tried to add before each list item one empty line more in source file, but unfortunately Documenter.jl interprets two consecutive empty lines as one, so nothing changes.

Reply:
Pity! But thanks for trying!